lib/passageidentity/version.rb

```diff
@@ -0,0 +1,5 @@
+# frozen_string_literal: true
+
+module Passage
+  VERSION = '0.2.4'
```

Comment:
I can put this back to `0.2.3` if preferred

Reply:
Yes, please leave this at version 0.2.3